android/src/play/java/com/dooboolab/RNIap/RNIapModule.kt

```diff
@@ -142,11 +138,8 @@ class RNIapModule(
                                 )
                             return@ConsumeResponseListener
                         }
-                        try {
-                            promise.resolve(true)
-                        } catch (oce: ObjectAlreadyConsumedException) {
-                            promise.reject(oce.message)
```

Comment:
This would be a crash

Reply:
The user would use the `try` and `catch` block on the JS side when it is a promise. Then it won't crash.

But I think we can accept this since it doesn't look like a big deal to me.